crates/driver/src/domain/competition/auction.rs

```diff
@@ -413,8 +416,19 @@ impl Price {
     }
 
     /// Apply this price to some token amount, converting that token into ETH.
+    ///
+    /// # Example: converting 1 ETH as token amount to eth::Ether
+    ///
+    /// Price of 1 ETH = 1e18 (since denominated in wei)
+    /// Token amount for 1 ETH = 1e18 (since denominated in wei)
+    ///
+    /// Normalized price which doesn't change the denominator of token amount is
+    /// a relative number and it has no unit. Normalized price = Price /
+    /// BASE = 1e18 / 1e18 = 1
+    ///
+    /// Ether amount = Token amount * Normalized price = 1e18 * 1 = 1e18
```

Comment:
Can you make this example executable (like https://doc.rust-lang.org/src/core/result.rs.html#760). I think then the whole text isn't needed

Reply:
added